### DISCUSSION TITLE
Terminal/Logbook 'read only' setting doesn't do anything.

### DISCUSSION THREAD

**Discussion by Ek-Videogames** [FakeFishGames/Barotrauma#14954] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When writing the 'book' item feature request, I noticed that the vanilla text documents are set to 'read only' in their terminal properties. However the unique UI-Style used by those documents removes the text entry box so it's not possible to type into them regardless.

However the logbook & the terminal use the normal UI-Style which does have a text entry box, and 'read only' doesn't actually seem to disable text entry it still allows commands to be entered like normal.

- (I'm guessing the 'intended' behavior for a terminal with read-only would be to have it only display it's wired input signal as normal, but not let the player type in and output signal.)

### Reproduction steps

- Enable 'read only' in a terminal or logboox.
![image](https://github.com/user-attachments/assets/0f806e6a-400c-4ad5-a563-80babdb74ce5)
- Type some stuff
![image](https://github.com/user-attachments/assets/33847cc1-9123-4fee-8b66-b86a517ba8bd)

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in our private repo, fix coming up in a future build!
